java/client/src/main/java/glide/api/models/commands/function/FunctionLoadOptions.java

```diff
@@ -0,0 +1,12 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands.function;
+
+/**
+ * Option for <code>FUNCTION LOAD</code> command.
+ *
+ * @see <a href="https://redis.io/docs/latest/commands/function-load/">redis.io</a>
+ */
+public enum FunctionLoadOptions {
```

Comment:
is it necessary to create an entire enum for a single value that's really only used as a constant string?  

Maybe, instead, add a constant string to ScriptAndFunctions interface?

Reply:
Not necessary, but I don't want to export service constants to a user[^1]. Is there another way?

[^1]: I want to hide already exported